posydon/popsyn/synthetic_population.py

```diff
@@ -224,8 +224,84 @@ def merge_parallel_runs(self, pop):
 # Helper classes #
 ##################
 
+class DFInterface:
```

Comment:
The class looks very general, hence we should consider to move this to the utils, e.g. if we'd like to use it for the grids, too.

Reply:
Fair. I have no real preference for either.

Reply:
What do you mean by interface? Functions that would apply to the population file, history, and oneline? It is super minor but this word seems a little opaque to me

Reply:
The `History` and `Oneline` classes use it to interface with the history and oneline Dataframes in the population file.